### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by GarvG**
I Completed Badges and Button! Please Review My PR. Suggestion are Highly Appreciated

### CODE REVIEW

File: Badge.css [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+}
+.IMG-XXL
+{
+    object-fit: cover;
+    border-radius: 50%;
+    height: 12rem;
+    width: 12rem;
+    position: relative;
+    margin: 10px;
+}
```
Comment (zoef-yt) [inline Badge.css]: Stay consistent with your code some places you are using rem and some places you are using px use one.

File: Badge.css [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+.badge-avatar--icon-XXL{
+    position: absolute;
+    background-color: var(--badgecolor);
+    right: 820px;
+    bottom: -135px;
+    border-radius: 50%;
+    padding: 12px;
+}
```
Comment (zoef-yt) [inline Badge.css:118]: spacing

File: Badge.css [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+.IMG-XL
+{
+    object-fit: cover;
+    border-radius: 50%;
+    height: 10rem;
+    width: 10rem;
+    position: relative;
+    margin: 10px;
+}
```
Comment (zoef-yt) [inline Badge.css:127]: spacing

**Review by zoef-yt — CHANGES_REQUESTED** [review]
Inconsistent spacing in code. use one unit either rem or px.

File: Badge.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Badge</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="sidebar">
+            <header>
+                My Components!
```
Comment (itsmebipul00) [inline Badge.html:18]: you could wrap this around a h1 tag

File: Badge.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                <li><a href="avatar.html">AVATAR</a></li>
+                <li><a href="alert.html">ALERT</a></li>
+                <li><a href="Badge.html">BADGE</a></li>
+                <li><a href="Button.html">BUTTON</a></li>
+                <li><a href="#">CARD</a></li>
+                <li><a href="#">IMAGE</a></li>
+                <li><a href="#">INPUT</a></li>
+            </ul>
+        </div>
+        <header class="head">
```
Comment (itsmebipul00) [inline Badge.html:30]: try to use only one header tag for a page

File: Badge.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+                        <p class="disable-badge--content">
+                            Button toggles badge visibility
+                        </p>
+                        <button class="disable-badge">
+                            HIDE
+                            <span class="disable-badge-icon">12</span>
+                        </button>
+                    </div>
+                    <h3 class="headers">DOCUMENTATION</h3>
+                    <img class="Doc-img" src="carbon-badge.png">
```
Comment (itsmebipul00) [inline Badge.html:59]: alt tags for all img tags

**Comment by itsmebipul00** [comment]
Good to go!!

**Review by zoef-yt — APPROVED** [review]
Good to go

**Pull request merged**